Dockerfile

```diff
@@ -1,5 +1,5 @@
-ARG DEBIAN_VERSION=11.6
-FROM "debian:$DEBIAN_VERSION-slim"
+ARG DEBIAN_VERSION=11
+FROM "mcr.microsoft.com/devcontainers/base:debian-$DEBIAN_VERSION"
```

Comment:
I switched to this base image because the primary use case for this image is devcontainers. 
https://github.com/devcontainers/images/tree/e63e378510a3b2a2b9bc1c73309ef2d3587f0ba1/src/base-debian